communication/marketing-team/README.md

```diff
@@ -1,57 +1,41 @@
-Future home for all things marketing team for contributor experience
-
-# Charter
-view our charter in this [marketing folder] for more about us and how to join
+The Contributor Marketing Team is part of the community-management subproject in
+[Contributor Experience] view our [charter] to learn more about us
 
 ## Purpose  
 to better inform the Kubernetes contributor community and highlight their work to
 the broader ecosystem
 
-# Core Marketing Team roles
+# Core Marketing Team
```

Comment:
```suggestion
## Core Marketing Team
```
It's odd having an H1 below an H2 like this.

Reply:
fixed